torch/csrc/Module.cpp

```diff
@@ -216,6 +217,14 @@ static PyObject* THPModule_crashIfATenASAN(PyObject* module, PyObject* arg) {
   return THPUtils_packInt32(at::_crash_if_asan(THPUtils_unpackInt(arg)));
 }
 
+static PyObject* THPModule_crashImmediately(
+    PyObject* module,
+    PyObject* noargs) {
+  throw std::runtime_error(
+      "Emulate uncaught exception that leads to the crash");
```

Comment:
It feels wrong to raise uncaptured exception from Python function, as API explicitly states that this is wrong. If you want to abort program execution, why not use `abort` or just raise a signal?
```suggestion
static PyObject* THPModule_abort(
    PyObject* module,
    PyObject* noargs) {
    abort();
```